app/models/person.rb

```diff
@@ -72,6 +73,10 @@ def latest_person_escort_record
     person_escort_records.where(status: 'confirmed').order(confirmed_at: :desc).first
   end
 
+  def latest_youth_risk_assessment
+    youth_risk_assessments.where(status: 'confirmed').order(confirmed_at: :desc).first
```

Comment:
attempting to dry this up looks more complicated, so I chose to keep as is

Reply:
I think you can use the auto generated scopes from the enum here, so this should work:

```ruby
  youth_risk_assessments.confirmed.order(confirmed_at: :desc).first
```

Reply:
Generated scopes don't work on a collection, but rather single records, so couldn't utilise it like this unfortunately. Im using the constant now however instead of the string